lte/cloud/go/services/lte/analytics/calculations/state.go

```diff
@@ -73,7 +73,7 @@ const (
 
 //GeneralMetricsCalculation ...
 type GeneralMetricsCalculation struct {
-	calculations.CalculationParams
+	calculations.BaseCalculation
```

Comment:
I don't understand / can you explain -- why did you create `BaseCalculation`, when its only current function is as an alias to `CalculationParams`

Reply:
Added it mainly to implement GetCalculationParams method.